package-res/resources/web/pentaho/visual/ccc/abstract/View.js

```diff
@@ -21,22 +21,24 @@ define([
   "cdf/lib/CCC/protovis",
   "../_axes/Axis",
   "../util",
+  "pentaho/util/object",
+  "pentaho/data/filter",
   "pentaho/visual/events",
   "pentaho/visual/color/utils",
   "pentaho/visual/color/paletteRegistry",
   "pentaho/data/TableView",
   "pentaho/i18n!view"
 ], function(View,
-    def, pvc, cdo, pv, Axis,
-    util, visualEvents, visualColorUtils, visualPaletteRegistry,
-    DataView, bundle) {
+            def, pvc, cdo, pv, Axis,
```

Comment:
If it were obviously an accident, I wouldn't complain. I'm kind of sick of commenting formatting issues when they are so many. These things, you should do them while you are coding, not after, or they will end up being missed. It's a matter of discipline. I didn't even comment on all the other "violations" in the remaining files. This one was the "drop"...

On the intention of the PR, on the programming side, I expect this to be almost final. On the docs and unit tests, I agree.